src/includes/sections/about.ejs

```diff
@@ -506,7 +506,7 @@
               </div>
               <div class="col p-4 d-flex flex-column position-static">
                 <strong class="d-inline-block mb-2">Article</strong>
-                <h3><a href="https://blog.percy.io/visual-testing-in-nightwatch-js-with-percy-1b68c122cf94"
+                <h3><a href="www.browserstack.com/guide/visual-regression-testing-in-nightwatchjs#toc3"
```

Comment:
Missing https://

Reply:
Sorry for underlooking and pushing these silly changes; actually, these changes are from my old PR. So, I copied from that PR without testing them again. 
I won't repeat this.